server.js

```diff
@@ -106,24 +98,10 @@ app.use(express.static(path.join(__dirname, "public")))
 app.use(apiLogger)
 
 // Routes layer setup
-app.use("/v1/auth", authRouter)
-app.use("/v1/sites", sitesRouter)
-app.use("/v1/sites", pagesRouter)
-app.use("/v1/sites", collectionsRouter)
-app.use("/v1/sites", collectionPagesRouter)
-app.use("/v1/sites", directoryRouter)
-app.use("/v1/sites", foldersRouter)
-app.use("/v1/sites", resourceRoomRouter)
-app.use("/v1/sites", resourcesRouter)
-app.use("/v1/sites", resourcePagesRouter)
-app.use("/v1/sites", imagesRouter)
-app.use("/v1/sites", documentsRouter)
-app.use("/v1/sites", mediaSubfolderRouter)
-app.use("/v1/sites", homepageRouter)
-app.use("/v1/sites", settingsRouter)
-app.use("/v1/sites", navigationRouter)
-app.use("/v1/sites", netlifyTomlRouter)
-app.use("/v1/user", usersRouter.getRouter())
+// To avoid refactoring auth router v1 to use dependency injection
+app.use("/v1/auth", authV2Router.getRouter())
+app.use("/v1/sites/:siteName", authenticatedSitesSubrouterV1)
+app.use("/v1", authenticatedSubrouterV1)
```

Comment:
could we add documentation so that the differences between all 3 are explicitly stated somewhere? i'm afraid that this might lead to confusion down the line, when we aren't as clear what these mean

Reply:
Makes sense, 00066be